Library/Formula/agda.rb

```diff
@@ -17,7 +18,11 @@ class Agda < Formula
     sha256 "d0e17808ee896cdedc499b010b67961dc3db3c430c501faef16c4b4482238447" => :mountain_lion
   end
 
-  head "https://github.com/agda/agda.git", :branch => "master"
+  head do
```

Comment:
And you're right about the issue you filed this morning; `head` lines should be merged into `head do` blocks when there's additional resources/deps/etc.